src/components/MainComponent.vue

```diff
@@ -1,169 +1,137 @@
+							<span class="fa fa-arrow-left" aria-hidden="true"/>Prev
+						</button>
+						<button :disabled="!URLAfter" @click="createRequestURL('next')" class="nav-pill">
+							Next<span class="fa fa-arrow-right" aria-hidden="true"/>
+						</button>
+					</li>
+				</ul>
+			</header>
+		</headroom>
+		<h1 class="photos-heading">Your pictures<span class="fa fa-picture-o" aria-hidden="true"/></h1>
+		<div class="photos-container">
+			<transition v-for="picture in pictures" :key="picture.url" name="slide-fade">
+				<img :src="picture" class="photos-container__photo">
+			</transition>
+		</div>
+		<error-display v-if="showError" :error-value="error" @close="showError = false"/>
+		<widget />
+	</div>
 </template>
 
 <script>
-import ErrorDisplay from './ErrorDisplay.vue'
-import CheatSheet from './CheatSheet.vue'
-import { headroom } from 'vue-headroom'
-import API from '../api'
+import { headroom } from 'vue-headroom';
+import ErrorDisplay from './ErrorDisplay.vue';
+import Widget from './Widget.vue';
+import API from '../api';
+
 export default {
-  name: 'MainComponent',
-  components: {
-    ErrorDisplay,
-    CheatSheet,
-    headroom
-  },
-  data () {
-    return {
-      pictures: [],
-      speed: 500,
-      subreddit: '',
-      error: '',
-      count: 0,
-      showError: false,
-      after: false,
-      before: false
-    }
-  },
-  methods: {
-    copyValue (value) {
-      this.subreddit = value
-      this.getURL(value)
-    },
-    checkFormat (url) {
-      return url.slice(url.length - 4, url.length - 3) === '.'
-    },
-    sendRequest (url) {
-      API.get(url)
-          .then(response => {
-            this.before = response.data.data.before
-            this.after = response.data.data.after
-            return response.data.data.children
-          })
-          .then(pictures => {
-            pictures.forEach((key) => {
-              if (this.checkFormat(key.data.url)) this.pictures.push(key.data.url)
-            })
-          })
-      .catch(error => {
-        this.error = error.message
-        this.showError = true
-        this.before = false
-        this.after = false
-      })
-    },
-    getURL (param) {
-      this.pictures = []
-      this.error = ''
-      this.showError = false
-      let url = ''
-      switch (param) {
-        case 'prev':
-          if (this.count % 5 === 0) this.count++
-          else this.count -= 25
-          url = `https://www.reddit.com/r/${this.subreddit}.json?count=${this.count}&before=${this.before}`
-          break
-        case 'next':
-          this.count += 25
-          url = `https://www.reddit.com/r/${this.subreddit}.json?count=${this.count}&after=${this.after}`
-          break
-        default:
-          url = `https://www.reddit.com/r/${this.subreddit}.json`
-      }
-      this.sendRequest(url)
-    }
-  }
-}
+	name: 'MainComponent',
+	components: {
+		ErrorDisplay,
+		Widget,
+		headroom
+	},
+	data() {
+		return {
+			error: '',
```

Comment:
Change the name to `errorMessage`. Consider passing this prop to the `ErrorDisplay` component.